src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovies } from '../../api/api';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const { addMovie } = props;
+
+  const [title, setTitle] = useState('');
+  const [foundMovie, showMovie] = useState<Movie | null>(null);
+  const [foundStatus, setFoundStatus] = useState(true);
+
+  const findMovie = async () => {
+    try {
+      const movie = await getMovies(title);
+
+      if (movie.Response === 'False') {
+        showMovie(null);
+        setFoundStatus(false);
+      } else {
+        showMovie(movie);
+        setFoundStatus(true);
+      }
+    } catch (error) {
+      showMovie(null);
+      setFoundStatus(false);
+    }
+  };
```

Comment:
` showMovie(null);
  setFoundStatus(false);`

no need to write this code twice